internal/instrument/instrument.go

```diff
@@ -303,16 +303,19 @@ func addInFunctionCode(list []dst.Stmt, tc *typechecker.TypeChecker, conf config
 				out = append(out, stmt)
 				appendStmt = false
 				out = append(out, ginMiddleware(stmt))
+				stmt.Decorations().Start.Prepend(dd_instrumented)
```

Comment:
Not sure about this, we already put these changes between `//dd:startwrap` and `//dd:endwrap`. Why do we need `//dd:instrumented`? And why it was added only to Gin, Echo, Chi and not the others ?

Reply:
@ahmed-mez I found out why, check my PR #42. Orchestrion applies two patterns: wrapping and instrumenting. The second one is the right one when we add code when a known statement is found, and the added code isn't actually wrapping the existing statement.

@knusbaum I think our PRs are going to clash. If you want to remove the `dd_instrumented` code, they'll merge fine.